hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HMobStore.java

```diff
@@ -139,7 +139,8 @@ public Configuration getConfiguration() {
    */
   @Override
   protected KeyValueScanner createScanner(Scan scan, ScanInfo scanInfo,
-      NavigableSet<byte[]> targetCols, long readPt) throws IOException {
+    NavigableSet<byte[]> targetCols, long readPt,List<KeyValueScanner> additionalScanners)
```

Comment:
Has to be a KVScanner? Can't be a CellScanner? (Took a look... seems like the latter is something else...)

Reply:
It Has to be a KVScanner. I will explain the reason in the following comment:
https://github.com/apache/hbase/pull/2498#discussion_r502654326